packages/muelu/test/tutorial/CMakeLists.txt

```diff
@@ -0,0 +1,383 @@
+INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR})
+INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR}/../unit_tests)
+
+# first ensure that these cmake boolean variables are defined
+ASSERT_DEFINED(
+  ${PACKAGE_NAME}_ENABLE_Epetra
+  ${PACKAGE_NAME}_ENABLE_Galeri
```

Comment:
I don't think `${PACKAGE_NAME}_ENABLE_Galeri` is defined in MueLu even when Galeri is built.